### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by AeryxSunyraX** [jrs0ul/nes_survival#48] [Q&A]
[https://drive.google.com/file/d/1jXibvvGuuBET411__ZeHS7o7dXE_dBJY/view?usp=sharing](url)
[https://drive.google.com/file/d/1-EztZ6da8wiU7cHLoW7fRppoK3QxWFk2/view?usp=sharing](url)

**Comment by jrs0ul** [comment]
I see two separate files. You need to combine them in to one(probably by hand), for the multiple music tracks to work. Also it's just two songs, the title music is a third track.
The beginning of your music.s file should have all three tracks like this:

> music_data_untitled:
> 	.byte 5
> 	.word @instruments
> 	.word @samples-52
> 	.word @song0ch0,@song0ch1,@song0ch2,@song0ch3,@song0ch4 ; 00 : Outside
> 	.byte .lobyte(@tempo_env_1_mid), .hibyte(@tempo_env_1_mid), 0, 0
> 	.word @song1ch0,@song1ch1,@song1ch2,@song1ch3,@song1ch4 ; 01 : Indoors
> 	.byte .lobyte(@tempo_env_1_mid), .hibyte(@tempo_env_1_mid), 0, 0
> 	.word @song2ch0,@song2ch1,@song2ch2,@song2ch3,@song2ch4 ; 02 : Title
> 	.byte .lobyte(@tempo_env_1_mid), .hibyte(@tempo_env_1_mid), 0, 0
> 	.word @song3ch0,@song3ch1,@song3ch2,@song3ch3,@song3ch4 ; 03 : GameOver
> 	.byte .lobyte(@tempo_env_1_mid), .hibyte(@tempo_env_1_mid), 0, 0
> 	.word @song4ch0,@song4ch1,@song4ch2,@song4ch3,@song4ch4 ; 04 : Howl
> 	.byte .lobyte(@tempo_env_1_mid), .hibyte(@tempo_env_1_mid), 0, 0
> 

What happened to your previous question? Did you deleted it ?

**Reply by AeryxSunyraX** [reply]
i was susspended and i created new account

**Comment by AeryxSunyraX** [comment]
When i copy and paste my chr data to main.chr with yy-chr then the game crashes and went into black screen

**Reply by jrs0ul** [reply]
Try to preserve the original 8x8 tile at the bottom right(it's a dot in a black background) of the sprite tile-set, I use it for the zero-sprite. Other than that I have no clue why it's happening.
